ci/vale/dictionary.txt

```diff
@@ -0,0 +1,1474 @@
+abc
+absecon
+abspath
+adc
+addon
+autovacuum
+addons
+addr
+addressfield
+adduser
+adminpack
+adodb
+aes
+ahci0
+ahvz
+ajp
+aker
+alives
+allmasquerade
+allnodes
+allrouters
+amavis
+amavisd
+amd64
+amongst
+analytics
+anonymize
+anonymizing
+ansible
+antispam
+ap
+apache2
+apha
+appimage
+appletalk
+appserver
+apxs
+aren
+argv
+arping
+aske
+async
+authdaemonrc
+authlib
+authmodulelist
+authmysql
+authmysqlrc
+authn
+authomator
+authy
+autocomplete
+autocompletion
+autoconfiguration
+autoconfigure
+autohosting
+autojoin
+autojump
+autologin
+autologout
+automagically
+automattic
+autorelabel
+autorun
+autoscaling
+autossh
+autostart
+auxprop
+avz
+awk
+awstats
+azc
+backend
+backends
+backported
+backporting
+backports
+backreference
+backreferences
+backtick
+bak
+bampton
+bandwdith
+bandwith
+barcode
+baremetal
+base64
+basearch
+bashdoor
+bashrc
+bc
+berkshelf
+bgrewriteaof
+binlog
+bitbucket
+bitrate
+blocklists
+boonex
+bootable
+bootloader
+bootup
+bootups
+bounceback
+brackley
+bram
+broswer
+bs4
+bugtracker
+bukkit
+bundler
+bungeecord
+burndown
+byrequests
+bzip2
+bzip2
+bzrtools
+c2s
+cabd
+cacert
+caddy
+caddyfile
+caker
+cakey
+calc
+captcha
+carootcert
+celeryd
+centos
+centos7
+ceph
+cer
+certbot
+certcheck
+cfg
+cgi
+changelog
+changelogs
+changeme
+changeset
+chatrooms
+cheatsheet
+checkboxes
+checkmark
+checksummed
+checksums
+chefspec
+chery
+chgrp
+chmod
+chown
+christoph
+chroot
+chrooted
+citad
+cjones
+clearspace
+cli
+clickboard
+clickjacking
+client1
+client2
+clojure
+clonefrom
+clozure
+cls
+cluebringer
+cluster1
+clustermgr
+clustermgrguest
+cmd
+cmdmod
+cmusieve
+cnf
+codebase
+codeblock
+codec
+com2sec
+comodo
+conda
+config
+configs
+configtest
+confluece
+convolutional
+coose
+copyfrom
+copyto
+coreutils
+corporateclean
+couldn
+cpanminus
+cpu
+cqlsh
+cqlshrc
+craftbukkit
+craigslist
+crashingdaily
+createdt
+createfromstackscript
+cron
+crond
+cronie
+cronjob
+cronjobs
+crontab
+crt
+crypto
+csr
+css
+csum
+csv
+ctools
+ctrl
+customizability
+customizable
+cyberduck
+cyg
+cygwin
+cz
+d39a
+d693
+daemonization
+daemonize
+daemonized
+daemonizes
+daemonizing
+dahdi
+datadir
+dataset
+datasets
+datasource
+datastax
+datastore
+datbase
```

Comment:
Should not be here. Fix in `/applications/cloud-storage/how-to-install-a-turtl-server-on-ubuntu.md`